<ui version="4.0" >
 <class>CfgSerialDialog</class>
 <widget class="QDialog" name="CfgSerialDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuraci√≥n Puerto Serial</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Dispositivo</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="comboPort" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Velocidad</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="comboSpeed" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Longitud de datos</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="comboSize" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Paridad</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="comboParity" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Bits de parada</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QComboBox" name="comboStop" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>XonXoff</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QComboBox" name="comboXonXoff" />
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>RtsCts</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QComboBox" name="comboRtsCts" />
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>DsrDtr</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QComboBox" name="comboDsrDtr" />
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>Timeout</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QDoubleSpinBox" name="spinTimeOut" >
     <property name="toolTip" >
      <string>Valores negativos hacen que no se tenga en cuenta el tiemeout.</string>
     </property>
     <property name="minimum" >
      <double>-1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CfgSerialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CfgSerialDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
